Do not change SONAME in Linux for VBoxOGL and VBoxEGL
--- ./src/VBox/Additions/common/crOpenGL/Makefile.kmk.orig	2016-10-14 03:28:39.435782415 +0100
+++ ./src/VBox/Additions/common/crOpenGL/Makefile.kmk	2016-10-14 03:28:59.434036287 +0100
@@ -245,7 +245,6 @@ endif
   	dl
   endif
  else
-  VBoxOGL_SONAME.linux = libGL.so.1
   VBoxOGL_LDFLAGS.linux += -Wl,-e,LibMain
  endif
 endif
@@ -731,7 +730,6 @@ ifndef VBOX_USE_SYSTEM_XORG_HEADERS
 VBoxEGL_INCS           = $(VBOX_PATH_X11_ROOT)/mesa-11.0.7
 endif
 VBoxEGL_LIBS           = $(VBOX_LIB_OGL) $(VBOX_LIB_IPRT_GUEST_R3_SHARED)
-VBoxEGL_SONAME.linux   = libEGL.so.1
 
 include $(FILE_KBUILD_SUB_FOOTER)
 
